### PR TITLE
`tensorflow`: add `tf.ones`, `tf.zeros`, `tf.zeros_like` and `tf.ones_like` functions

### PR DESCRIPTION
**Pull request by hoel-bagard**
Taken from [here](https://github.com/hmc-cs-mdrissi/tensorflow_stubs/blob/main/stubs/tensorflow/__init__.pyi#L584) and updated to tensorflow 2.15.

### CODE REVIEW

**Comment by hoel-bagard** [comment]
@hmc-cs-mdrissi If you have time to check.

File: stubs/tensorflow/tensorflow/experimental/dtensor.pyi [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from _typeshed import Incomplete
```
Comment (hmc-cs-mdrissi) [inline stubs/tensorflow/tensorflow/experimental/dtensor.pyi:1]: This file could use `def __getattr__(name: str) -> Incomplete`.

**Review by hmc-cs-mdrissi — APPROVED** [review]
One small comment, otherwise looks good.

**Comment by JelleZijlstra** [comment]
Could you solve the merge conflict? Then I'll merge.

**Comment by hoel-bagard** [comment]
@JelleZijlstra I've solved the merge conflict.

**Review by JelleZijlstra — APPROVED** [review]

**Pull request merged**